pkg/kubernetes/client/client.go

```diff
@@ -90,9 +95,12 @@ func (cl *Client) GetJivaVolume(name string) (*jv.JivaVolume, error) {
 	ns := "openebs"
```

Comment:
We should not be havaing any namespace hardcoded, get the namespace using an ENV will be a better idea

Reply:
this is default value, we are taking the namespace from storageclass